Flask_app/services/auth.py

```diff
@@ -29,4 +28,12 @@ def create_user(model: Any,
 
 def get_user_by_email(email):
     user = User.query.filter_by(email=email).first()
-    return user
\ No newline at end of file
+    return user
+
+
+def validate_email(email):
+    pattern = r'^[\w\.-]+@[\w\.-]+\.\w+$'
+    if re.match(pattern, email):
+        return True
+    else:
```

Comment:
You can remove else,
Just return it outside if block.

It's better to write else block if you are not returning inside your if block, But since you are returning it, remove else